src/devices/ambient_weather_tx8300.c

```diff
@@ -0,0 +1,136 @@
+/* Ambient Weather TX-8300 Thermometer
```

Comment:
The device should be named ambientweather_tx8300 -- We'll slowly move to just one word for manufacturer and one for device to simplify parsing in consumers.